Foundation/NSNumber.swift

```diff
@@ -198,14 +198,20 @@ extension NSNumber : ExpressibleByFloatLiteral, ExpressibleByIntegerLiteral, Exp
 
 }
 
+@_silgen_name("_CFNumberGetType2")
```

Comment:
What's the difference between `ForSwiftFoundationOnly.h` and `ForFoundationOnly.h`?
It seems `_CFNumberInit*()` functions are placed in `ForFoundationOnly.h`.
Should `_CFNumberGetType2` also be placed in `ForFoundationOnly.h`?